Assets/Engine/Inventory/Extensions/TakeStoreInteraction.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SS3D.Content.Furniture;
+using SS3D.Engine.Interactions;
+using SS3D.Engine.Interactions.Extensions;
+using SS3D.Engine.Inventory;
+using SS3D.Engine.Inventory.Extensions;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Content.Systems.Interactions
+{
+    public class Store : StoreInteraction { };
```

Comment:
Please use `using Store = SS3D.Content.Systems.Interactions.StoreInteraction`

Reply:
I'll have to change the StoreInteraction so you can only store a Power Cell on it

Reply:
It's perfectly fine then.